reg_tests/CTestList.cmake

```diff
@@ -242,8 +199,6 @@ if(NOT ENABLE_CUDA)
   #=============================================================================
   # Performance tests
   #=============================================================================
-  add_test_p(uqSlidingMeshDG 8)
```

Comment:
We can also remove the performance test macros from the top of this file unless we want to replace them with more targeted wind energy edge variants. I'm open to either option.

Reply:
@psakievich I was hoping to add wind-energy specific performance tests here. The empty section header was meant as reminder for whoever that ends up adding these tests later.

Reply:
Sounds good to me